company/okrs/index.md

```diff
@@ -0,0 +1,92 @@
+
+## Quarterly OKRs
+
+Our OKRs are public:
+
+- [2019-Q4 (active)](2019_q4.md)
+- [2019-Q3](2019_q3.md)
+
+## What are OKRs?
+
+[Objectives and Key Results (OKRs)](https://en.wikipedia.org/wiki/OKR) is a specific way of setting goals that we use at Sourcegraph.
+
+- For each goal (called an OKR), you state an **objective** (what you're aiming to do) and **key results** (how you'll measure success toward the objective).
+- OKRs form a "tree". We have a few broad company-level OKRs. Each company-level OKR has many sub-OKRs, for teams whose work contributes up to the company-level OKR. This codifies why/how each team's work is important to the company.
+- We are not yet doing personal OKRs company-wide. This document is only about company and team OKRs. (Some people are experimenting with personal OKRs with their manager. If that works well, we may expand it, and this page will be updated.)
+
+Our OKR practices are heavily inspired by [GitLab's OKR practices](https://about.gitlab.com/company/okrs/).
+
+### OKRs are stretch goals
+
+1. Set OKRs to be ambitious but achievable, so that they convey your aims and what you could achieve if things go well.
+  - In (rough) numeric terms, if you achieve less than 70% of your KR, it was probably not achievable. If you regularly achieve 100%+ of your KRs, they are probably not ambitious enough.
+  - Your KRs are **not** the 99.99% likelihood outcome or the 1% moonshot outcome.
+1. OKRs are for communication and alignment (everyone knowing what's important, why, and how it'll get done), **not** for estimation, scheduling, or promising/committing.
+   - Setting an OKR for *X* is not the same as *committing* to shipping *X* to customers. We are much more conservative in setting expectations with customers about ship dates than we are in setting OKRs, and it wouldn't make sense to be locked into a ship date that we decided at the beginning of the quarter when setting OKRs.
+   - OKRs are not used directly for giving individual performance review feedback or for compensation purposes.
+   - Write your OKRs so they communicate effectively to an audience who understands that OKRs aren't promises. Don't write OKRs so cautiously that they are hard to understand. (For example, "Improve code intelligence support for Java" is a good objective, especially with well defined key results. But "Investigate feasibility of improvements to Java code intelligence" is probably bad.)
+1. For new projects, it can be hard to define the right OKRs. Do your best. Then refine them next quarter.
+
+## Setting OKRs
+
+### Schedule
+
+We set OKRs for each quarter. The (rough) schedule for setting OKRs for an upcoming quarter is:
+
+- -3 weeks: CEO creates the upcoming quarter's OKR document with company-level goals (as a Google Doc for easy collaboration) and links it from this page.
+- -1 week: Each Sourcegraph team has written their team OKRs, and these OKRs have been approved by the owners of the OKRs above them.
```

Comment:
This doc is missing how we get from -3 weeks to -1 week. I would frame these as deadlines in the prose above.

```suggestion
- -2 weeks: Each Sourcegraph team adds proposed OKRs to the doc and solicits feedback from relevant stakeholders.
- -1 week: OKRs have been approved by the owners of the OKRs above them.
```